app/models/concerns/has_recent_items.rb

```diff
@@ -13,8 +13,30 @@ def recent_item_types
       self.class.recent_item_types
     end
 
+    def viewable_map
+      self.class.viewable_map
+    end
+
     def recent_items
-      sorted_recent_item_links.preload(:item).map(&:item).compact
+      recent_item_links.order(updated_at: :desc).preload(:item).map(&:item).compact
+    end
+
+    def viewable_recent_items
+      viewable_recent_item_links.order(updated_at: :desc).preload(:item).map(&:item).compact
+    end
```

Comment:
Do we need both of these methods? I don't see that `recent_items` is ever called and I don't have a use-case for it.

Reply:
Can we replace all this with 
```
    def recent_items
      viewable_recent_item_links.order(updated_at: :desc).preload(:item).map(&:item).compact
    end
```

Reply:
There is no use-case for unfiltered items

Reply:
Fixed